### PR TITLE
Makes coverage chips colour percentage gradient

### PR DESCRIPTION
**Pull request by KiyaDoig**

### CODE REVIEW

**Comment by MurrayJack** [comment]
there seems to be an issue with the padding on the chips
https://deploy-preview-12--starlux-team.netlify.com/

File: app/src/components/projects.jsx [hunk 1]
```diff
@@ -88,7 +99,7 @@ const StyledPercentage = styled.div`
 
 const StyledPercentageBad = styled.div`
     display: inline-block;
-    background-color: darkred;
+    background-color: #cc3232;
```
Comment (MurrayJack) [inline app/src/components/projects.jsx]: I think the orange is off here

File: app/src/components/projects.jsx [hunk 2]
```diff
@@ -73,9 +73,20 @@ const StyledVersion = styled.li`
...
+const StyledPercentageGood = styled.div`
     display: inline-block;
-    background-color: green;
+    background-color: #50B83C;
+    color: white;
+    line-height: 1.4em;
+    margin: 4px;
+    border-radius: 3px;
+    font-size: 14px;
+    padding: 4px 8px;
```
Comment (MurrayJack) [inline app/src/components/projects.jsx]: i think the padding, is off here

**Review by MurrayJack — COMMENTED** [review]
Hey!

**Comment by KiyaDoig** [comment]
:D !

**Comment by KiyaDoig** [comment]
Pretty happy with it only thought is maybe looks too green at coverage values we consider too low - could tweak it if you think so too

**Review by MurrayJack — APPROVED** [review]

**Comment by MurrayJack** [comment]
Cool, ok you can merge this

**Pull request merged**